pkg/csi/service/wcp/controller.go

```diff
@@ -144,10 +144,12 @@ func (c *controller) Init(config *cnsconfig.Config, version string) error {
 			return err
 		}
 	}
+	improvedTasksListViewEnabled := commonco.ContainerOrchestratorUtility.IsFSSEnabled(ctx,
+		common.ListViewPerf)
```

Comment:
I do not see a feature gate added to supervisor YAML in this file.
Do you want to limit the change to Vanilla flavor only?

Reply:
I previously considered having the changes for all flavors in the same PR but I'll keep it separate so it's easy to revert if required. I'll create separate PRs for WCP/TKG which will include the feature gate in their yamls.